src/components/Chip/styled/container.js

```diff
@@ -49,6 +53,20 @@ const StyledContainer = attachThemeAttrs(styled.span)`
             color: ${props.palette.getContrastText(props.palette[props.variant].main)};
         `};
 
+        ${props =>
+            props.size === 'large' &&
+            `
+                font-size: ${FONT_SIZE_TEXT_LARGE};
+                height: 2.9rem;
+            `};
+    
+        ${props =>
+            props.size === 'small' &&
+            `
+                font-size: ${FONT_SIZE_TEXT_SMALL};
+                height: 1.6rem;
+            `};
+
     ${props =>
```

Comment:
We should use different padding when the size is "large" and "small"
large: 1rem
small: 0.5rem